vshard/storage/init.lua

```diff
@@ -2825,6 +2845,7 @@ local function rebalancer_service_f(service)
     local module_version = M.module_version
     while module_version == M.module_version do
         service:next_iter()
+        M.rebalancer_ratelimit:flush()
```

Comment:
The rebalancer can sleep for an hour. It means any errors suppressed during the last 30 seconds before the rebalancer goes into idle mode are not going to be flushed. Or would be flushed much much later.

Which actually might be a problem. The flush right now doesn't print the timestamps of the original logs when they started being suppressed, right? So here when the rebalancer wakes up an hour later and prints some errors being suppressed, we won't know that they happened an hour ago.

Reply:
That's a very strong point and the main motivation, why I introduced the background fiber for flushing the ratelimiters.